web/client/plugins/manager/GroupManager.jsx

```diff
@@ -88,8 +95,93 @@ class GroupManager extends React.Component {
  * Allows an administrator to browse user groups.
  * Renders in {@link #plugins.Manager|Manager} plugin.
  * @name GroupManager
+ * @property {object[]} [attributeFields] attributes that should be shown in attributes tab of group manager.
```

Comment:
please specify defaults for attribute fields, like separator ecc

Reply:
```suggestion
 * @property {object[]} [attributeFields] attributes that should be shown in attributes tab of group manager. By default this array contains one `notes` attribute with `controlType`: `text`. Every object in this array can contain: 
 * - `name`: the name of the attribute
 * - `title`: the string to show as label for the attribute.  If not present 
 * - `controlType`: The input control to use. can be : `string`, `text` (text area), `date` (date picker) and `select`. 
 * - `controlAttributes`: attributes specific of the `controlType`. See the examples for more details.
```